dev.sh

```diff
@@ -7,15 +7,38 @@ PORT=${DOCKERPORT:-80}
 DB_READY=0
 TRIES=0
 MAX_TRIES=12
+export SOLR=4.10.4
 
 docker stop open-oni-dev || true
 docker rm open-oni-dev || true
 
+# start_container $1 = name of container, $2 = container running status
+start_container () {
```

Comment:
I think this function could be improved by checking the status if $2 is not passed in, but I don't know if that's necessary for this short dev script.  Thoughts?